style.css

```diff
@@ -63,6 +63,47 @@ header {
   height: auto;
 }
 
+#mobile-menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  padding-top: 16%;
+  padding-left: 7%;
+  display: none;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(96, 112, 255, 0.8);
+  z-index: 999;
+}
```

Comment:
- [ ] Great job so far @Porkalaikarthika and @riskmaris there are just a few changes you need to make and you'll be on your way to the next milestone :wink: 
- [ ] Your mobile menu works as expected good job with that :clap: . However, per the project requirements, you must stick to the design as much as possible (e.g., font, colors, images, text, margins) of [your chosen Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [ ] From the images below, you can see that the background of your design is missing the `blur` effect as seen in the Figma design, and because of this, the text in the background is overlapping with that of the `mobile menu items` and this is not ideal for user experience so I will suggest that you fix this so as to follow the Figma design closely :wink: 

| Your Design | The Figma |
|--------|--------|
| ![image](https://github.com/Porkalaikarthika/Portfolio/assets/102699958/7eed258b-f66c-4a0c-a1d5-80e0cfac7382) |![image](https://github.com/Porkalaikarthika/Portfolio/assets/102699958/0188185e-fa5e-445a-8862-c4d68a2a6913)|